models/contact_views/contactview_metadata.sql

```diff
@@ -0,0 +1,28 @@
+{{
+    config(
+        materialized = 'view',
+        indexes=[
+            {'columns': ['contact_uuid']},
+            {'columns': ['parent_uuid']},
+            {'columns': ['type']},
+            {'columns': ['uuid']},
+        ]
+    )
+}}
+
+SELECT
+    raw_contacts.doc ->> '_id'::text AS uuid,
+    raw_contacts.doc ->> 'name'::text AS name,
```

Comment:
Nit: we don't need the raw_contacts prefix since we are only querying a single table